web/static/vendor/maps.js

```diff
@@ -0,0 +1,101 @@
+function initMap() {
```

Comment:
If we aren't planning on customizing the functionality too much, I think it is fine.